migrations/versions/7805dfde3ca9_.py

```diff
@@ -0,0 +1,28 @@
+"""empty message
+
+Revision ID: 7805dfde3ca9
+Revises: ac76a3ecb338
+Create Date: 2021-03-10 11:25:26.878848
+
+"""
+from alembic import op
+import sqlalchemy as sa
+from sqlalchemy.dialects import postgresql
+
+# revision identifiers, used by Alembic.
+revision = '7805dfde3ca9'
+down_revision = 'ac76a3ecb338'
+branch_labels = None
+depends_on = None
+
+
+def upgrade():
+    # ### commands auto generated by Alembic - please adjust! ###
+    op.add_column('systems', sa.Column('rule_hit_details', postgresql.JSONB(astext_type=sa.Text()), nullable=True))
```

Comment:
Will it be good to have a separate table to store rule hit details?
i.e with columns - id, system_id, account_id, rule_id, rule_hit_details

Reply:
Hello, what should be the rule_id? looking at [this](https://github.com/RedHatInsights/ros-backend/pull/46/files#diff-a0e4b82fb8caf907a6fdd12040acaeea35910491cb3e7a7c038e99b94d809822R228), I think rule_id will be dynamic and only part "cloud_instance_ros_evaluation" will be constant.

Reply:
ahh okies! For every report, it has different `rule_id` &  you are storing all reports in one column, right? I think just ignore `rule_id` for now.